src/Network/Network/FirewallPolicy/FirewallCustomRule/AzureApplicationGatewayFirewallCustomRuleBase.cs

```diff
@@ -68,6 +88,11 @@ public override void ExecuteCmdlet()
             {
                 this.State = "Enabled";
             }
+
+            if (!this.MyInvocation.BoundParameters.ContainsKey("RateLimitThreshold"))
```

Comment:
Why is this needed? Was this to get around the Swagger issue? If so, can this be removed now?

Reply:
yes, it was used to get around swagger, will remove it